extensions/resource-deployment/notebooks/bdc/2019/deploy-bdc-existing-aro.ipynb

```diff
@@ -0,0 +1,350 @@
+{
+    "metadata": {
+        "kernelspec": {
+            "name": "python3",
+            "display_name": "Python 3"
+        },
+        "language_info": {
+            "name": "python",
+            "version": "3.6.6",
+            "mimetype": "text/x-python",
+            "codemirror_mode": {
+                "name": "ipython",
+                "version": 3
+            },
+            "pygments_lexer": "ipython3",
+            "nbconvert_exporter": "python",
+            "file_extension": ".py"
+        }
+    },
+    "nbformat_minor": 2,
+    "nbformat": 4,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "![Microsoft](https://raw.githubusercontent.com/microsoft/azuredatastudio/master/src/sql/media/microsoft-small-logo.png)\n",
```

Comment:
I mentioned this to Arvind - but we really shouldn't be using this particular image IMO since there's no indication that it's used outside of that context and so could be deleted at any time. I'd prefer to see it copied into the extensions that need it and have that copy referenced instead.